aiida/cmdline/commands/cmd_code.py

```diff
@@ -175,33 +175,29 @@ def show(code, verbose):
     table.append(['Prepend text', code.get_prepend_text()])
     table.append(['Append text', code.get_append_text()])
 
-    if verbose:
+    if VERDI_LOGGER.level <= LOG_LEVELS['DEBUG']:
         table.append(['Calculations', len(code.get_outgoing().all())])
 
     click.echo(tabulate.tabulate(table))
 
 
 @verdi_code.command()
 @arguments.CODES()
-@options.VERBOSE()
 @options.DRY_RUN()
 @options.FORCE()
 @with_dbenv()
-def delete(codes, verbose, dry_run, force):
+def delete(codes, dry_run, force):
     """Delete a code.
 
     Note that codes are part of the data provenance, and deleting a code will delete all calculations using it.
     """
     from aiida.manage.database.delete.nodes import delete_nodes
 
-    verbosity = 1
     if force:
-        verbosity = 0
-    elif verbose:
-        verbosity = 2
+        VERDI_LOGGER.setLevel('CRITICAL')
```

Comment:
Should `force=True` be interpreted as suppress all logging?

Reply:
I'm just replicating the previous behavior here...